.gitignore

```diff
@@ -23,6 +23,12 @@
 /test/**/*.so
 /test/expressions/*.cpp
 
+# benchmarks
+/benchmarks/*
+!/benchmarks/*.*
+benchmarks/*.cpp
```

Comment:
@t4c1 this scheme is what we do for the test executables so should just ignore the binaries, .cpp files, etc